frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsHours/FeatureMetricsHours.tsx

```diff
@@ -17,13 +17,15 @@ const StyledTitle = styled('h2')(({ theme }) => ({
 interface IFeatureMetricsHoursProps {
     hoursBack: number;
     setHoursBack: (value: number) => void;
+    label?: ReactNode;
 }
 
 export const FEATURE_METRIC_HOURS_BACK_DEFAULT = 48;
 
 export const FeatureMetricsHours = ({
     hoursBack,
     setHoursBack,
+    label = <StyledTitle>Period</StyledTitle>,
```

Comment:
allowing to disable the label by passing undefined. In the feature metrics the label is needed but in personal dashboard it's not.

Reply:
Of course, the input's been around for a while, probably, but we can still fix it up now that we're here 💁🏼